tests/beta/test_default_tensor.py

```diff
@@ -894,31 +894,21 @@ def circuit(*x):
             x = np.random.random([op.num_params])
             circuit(*x)
 
-    @pytest.mark.parametrize("observable", set(qml.ops.cv.obs))
-    def test_unsupported_observable_error(self, rep, observable):
+    def test_unsupported_observable_error(self, rep):
```

Comment:
This is another bad pattern: to test an error message we loop over a big list of operations that produce the error, instead of one!

Reply:
so much cleaner!